src/js/components/text/__tests__/KeptTextForm-test.jsx

```diff
@@ -35,39 +37,41 @@ describe("KeptTextForm", function() {
                                 {target: {value: "# world"}});
       TestUtils.Simulate.submit(form);
 
-      expect(fakeCreate).toBeCalledWith({
+      expect(fakeCreate.calledWith({
         id: null,
         type: "text",
         title: "Hello",
         text: "# world"
-      });
+      })).to.be.true;
     });
 
     it("should update an existing text", function() {
       var comp = TestUtils.renderIntoDocument(
         <KeptTextForm data={existingEntry} create={fakeCreate} update={fakeUpdate} />);
-      var form = comp.getDOMNode().querySelector("form");
 
-      TestUtils.Simulate.change(form.querySelector("input[type=text]"),
+      TestUtils.Simulate.change(reactDom.findDOMNode(comp.refs.title) ,
                                 {target: {value: "Plip"}});
-      TestUtils.Simulate.change(form.querySelector("textarea"),
+      TestUtils.Simulate.change(reactDom.findDOMNode(comp.refs.text),
                                 {target: {value: "# plip"}});
-      TestUtils.Simulate.submit(form);
+      TestUtils.Simulate.submit(reactDom.findDOMNode(comp.refs.title).form);
 
-      expect(fakeUpdate).toBeCalledWith({
+      expect(fakeUpdate.calledWith({
         id: 1,
         type: "text",
         title: "Plip",
         text: "# plip"
-      });
+      })).to.be.true;
     });
   });
 
   describe("#render", function() {
     it("should render HTML content", function() {
-      var comp = TestUtils.renderIntoDocument(<KeptTextForm data={existingEntry} />);
+      var renderer = TestUtils.createRenderer();
+      renderer.render(<KeptTextForm data={existingEntry} />);
+
+      var result = renderer.getRenderOutput();
 
-      expect(comp.getDOMNode().querySelector("form")).toBeTruthy();
+      expect(result.props.children.length).to.equal(3);
```

Comment:
Hmm that's not quite the same test; could we check that a `form` tag is present instead?